Servers/UI/OJS.Servers.Ui/Controllers/Api/UsersController.cs

```diff
@@ -25,6 +26,7 @@ public UsersController(IUsersBusinessService usersBusiness)
     /// <param name="username">The username of the user.</param>
     /// <returns>User profile info model.</returns>
     [HttpGet]
+    [AllowAnonymous]
```

Comment:
If no user is logged in then the endpoint will return only the username if it exists. This is done because the FE needs to know if the users profile exists at all, since even unauthorized users should be able to see profiles. If no profile exists, then the endpoint will return an error and the FE will know if any profile should be displayed. If the user is not logged in but the profile exists, then only the Username will be returned. Check out the logic in https://github.com/SoftUni-Internal/OpenJudgeSystem/blob/profilepage-redesign/Services/UI/OJS.Services.Ui.Business/Implementations/UsersBusinessService.cs#L30